pages.ca/common/cmatrix.md

```diff
@@ -0,0 +1,20 @@
+# cmatrix
+
+> Mostra una pantalla similar a la de Matrix en la terminal.
+> Més informació: <https://github.com/abishekvashok/cmatrix>.
+
+- Activa el desplaçament asíncron:
+
+`cmatrix -a`
+
+- Mostra el text en vermell:
+
+`cmatrix -C {{red}}`
+
+- Activa el mode multicolor:
```

Comment:
Why you don't use the translation of `rainbow`?

Reply:
I don't use the translation of `rainbow` because in catalan it translates to `arc de Sant Martí` and it's only used to talk about the meteorological phenomenon.